src/migration.types.ts

```diff
@@ -171,3 +171,11 @@ export type MigrationFunction<
     doesReportIncludeStacks: boolean;
   },
 ) => void;
+
+/**
+ * The behavior to have when an error occurs during the migration of an item:
+ * - "keep-original": if anything goes wrong during the migration of an item, the CLI `from-version` of the item is kept.
+ * - "keep-latter": if anything goes wrong during the migration of an item between n-version and n+1-version, the n-version of the item is kept.
```

Comment:
```suggestion
 * - "keep-last-successful-version": the version of the item obtained after the p-1 first successful steps is kept, and the n-p remaining steps are not applied.
```
"stop" would be a simpler alternative, but I am afraid it could be misread as something global, as opposed to just one item.

Not a strong opinion though if you prefer "stop".

Reply:
Yeah, I'm not a fan of `"stop"`, as you said it's too generic, and could be misleading.
I find `"keep-last-successful-version"` a bit long, what do you think about more simply `"keep-last-successful"`?

Reply:
It's an adjective so I would find it a bit weird: in the eyes of a consumer of the CLI I would wonder "last successful what ?"

Reply:
Yeah, it's fair enough. Happy to go for `"keep-last-successful-version"` then, I can't think of anything better.